src/constants.ts

```diff
@@ -94,7 +94,9 @@ export const MIME_TYPES = {
   excalidrawlib: "application/vnd.excalidrawlib+json",
   json: "application/json",
   svg: "image/svg+xml",
+  "excalidraw.svg": "image/svg+xml",
   png: "image/png",
+  "excalidraw.png": "image/png",
```

Comment:
this is unused and probably will always be, so let's remove

Reply:
wait, it's used for types somewhere

Reply:
Yep, the compilation did break until I changed it

Reply:
ok, let's leave it then